service/src/main/java/uk/gov/hmcts/reform/fpl/service/ApplicantMigrationService.java

```diff
@@ -0,0 +1,150 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.enums.PartyType;
+import uk.gov.hmcts.reform.fpl.model.Applicant;
+import uk.gov.hmcts.reform.fpl.model.ApplicantParty;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.OldApplicant;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.utils.PBANumberHelper;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import static com.google.common.base.Strings.isNullOrEmpty;
+import static java.util.stream.Collectors.toList;
+import static uk.gov.hmcts.reform.fpl.utils.PBANumberHelper.updatePBANumber;
+import static uk.gov.hmcts.reform.fpl.utils.PBANumberHelper.validatePBANumber;
+
+@Service
+public class ApplicantMigrationService {
+
+    @Autowired
+    private final ObjectMapper mapper = new ObjectMapper();
+
+    public String setMigratedValue(CaseData caseData) {
+        if (caseData.getApplicants() != null || caseData.getApplicant() == null) {
+            return "Yes";
+        } else {
+            return "No";
+        }
+    }
+
+    public List<Element<Applicant>> expandApplicantCollection(CaseData caseData) {
+        if (caseData.getApplicants() == null) {
+            List<Element<Applicant>> populatedApplicant = new ArrayList<>();
+
+            populatedApplicant.add(Element.<Applicant>builder()
+                .value(Applicant.builder()
+                    .party(ApplicantParty.builder()
+                        .partyId(UUID.randomUUID().toString())
+                        .build())
+                    .build())
+                .build());
+
+            return populatedApplicant;
+        } else {
+            return caseData.getApplicants();
+        }
+    }
+
+    public AboutToStartOrSubmitCallbackResponse validateAndUpdatePBANumbers(CaseDetails caseDetails) {
```

Comment:
validateAndUpdatePbaNumbers has nothing to do with ApplicantMigrationService. It should be in a new service

Reply:
@swalker125 still not a fan of splitting this Sam?